pychassisctl/chassis_control.py

```diff
@@ -67,6 +67,11 @@ def __init__(self, bus, name):
                 'object_name': '/org/openbmc/settings/host0',
                 'interface_name': 'org.freedesktop.DBus.Properties'
             },
+            'control': {
```

Comment:
host_control would be a little more descriptive here.  Otherwise LGTM ( my last comment, I swear ).